src/data/blog/portfolio-changelog-2026.md

```
@@ -203,3 +220,24 @@ description: A comprehensive nadzu.me Portfolio Changelog documenting updates fo
 [PR8]: https://github.com/nxdun/Portfolio/pull/8
 [PR10]: https://github.com/nxdun/Portfolio/pull/10
 [PR11]: https://github.com/nxdun/Portfolio/pull/11
+[PR12]: https://github.com/nxdun/Portfolio/pull/12
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "pr12"

(MD053, link-image-reference-definitions)